### ISSUE TITLE
Couldn't Build and Sync on Android

### ISSUE THREAD

**Issue by dewelloper** [invertase/react-native-firebase-starter#104]
Iam new in this project!
in my case windows 10 Android studio and problem is that.
I have followed the instruction i have downloaded the google settings json from firebase changed the gradle but it does not sync, giving me this errors:

ERROR: The library com.google.android.gms:play-services-measurement-base is being requested by various other libraries at [[16.0.0,16.0.0], [16.0.5,16.0.5]], but resolves to 16.0.5. Disable the plugin and check your dependencies tree using ./gradlew :app:dependencies.

should i disable or find and why where :)

**Comment by petixiuxx** [comment]
I've experienced the same problem in Android too

**Comment by petixiuxx** [comment]
Hey @dewelloper , Have you found any solution ? In my case, I've started Android Studio to sync and everything working fine

**Comment by jedabero** [comment]
I get a similar error in my configuration:
```
Cause 1: org.gradle.api.GradleException:
The library com.google.android.gms:play-services-measurement-base is
being requested by various other libraries at [[16.0.5,16.0.5]], but resolves to 16.4.0.
Disable the plugin and check your dependencies tree using ./gradlew :app:dependencies.
```
Plus I also get this:
```
Cause 2: org.gradle.api.GradleException:
The library com.google.firebase:firebase-analytics is
being requested by various other libraries at [[16.0.6,16.0.6]], but resolves to 16.4.0.
Disable the plugin and check your dependencies tree using ./gradlew :app:dependencies.
```

An apparent solution I found out there was to upgrade all firebase and google dependencies, but that only solved for `firebase-analytics`, actually just upgrading `firebase-analytics` to 16.4.0 solves that. But the problem with `play-services-measurement-base` still occurs.

I should also mention that this doesn't happen on a fresh install of the library.

I've tried installing my dependencies one by one but I get different errors (I have `jumboMode` enabled in `dexOptions` due to https://github.com/blackuy/react-native-twilio-video-webrtc requiring it)

**Comment by mikehardy** [comment]
This used to be a problem but with updated versions (PR here https://github.com/invertase/react-native-firebase-starter/pull/118) everything comes together fine
